apps/web/app/features/stacking/components/stacking-info-card.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMemo } from 'react';
+
+import { HStack, VStack, styled } from 'leather-styles/jsx';
+
+import { Button, CheckmarkCircleIcon, CircleIcon } from '@leather.io/ui';
+
+export interface StackingInfoCardProps {
+  poolAmount: string;
+}
+
+export function StackingInfoCard({ poolAmount }: StackingInfoCardProps) {
+  const confirmations = useMemo(
+    () => [
+      {
+        id: 'terms-and-conditions',
+        text: 'I have read and accepted the pool’s terms and conditions',
+        accepted: true,
+      },
+      {
+        id: 'interact-wallet',
+        text: 'Allow the pool contract to interact with your wallet',
+        accepted: false,
+      },
+      {
+        id: 'confirm',
+        text: 'Confirm and start pooling',
+        accepted: false,
+      },
```

Comment:
this is a static obj so can be outside component, no need for usememo

Reply:
hm, i can create sep variable where put accepted indexes though